### ISSUE TITLE
create a new release

### ISSUE THREAD

**Issue by christopherpickering** [awebdeveloper/pre-commit-stylelint#3]
Hey, do you mind creating a new release to point pre-commit to? The last release was 2017, so I leave the release blank in the config.. but hoping to avoid this warning in my pre-commit:

```sh
[WARNING] The 'rev' field of repo 'https://github.com/awebdeveloper/pre-commit-stylelint' appears to be a mutable reference (moving tag / branch).  Mutable references are never updated after first install and are not supported.  See https://pre-commit.com/#using-the-latest-version-for-a-repository for more details.  Hint: `pre-commit autoupdate` often fixes this.
```

thanks!

**Comment by christopherpickering** [comment]
although I suppose we can do this and use a recent commit sha... as the docs point out.

```yml
 - repo: https://github.com/awebdeveloper/pre-commit-stylelint
    rev: '8f63da4'
    hooks:
      - id: stylelint
```

**Comment by thibaudcolas** [comment]
In case anyone needs this, I put together an alternative hook that automatically syncs stylelint releases: https://github.com/thibaudcolas/pre-commit-stylelint. So you can just write the stylelint version you’re after:

```yaml
- repo: https://github.com/thibaudcolas/pre-commit-stylelint
  rev: v14.4.0
  hooks:
    - id: stylelint
```

Or with additional dependencies:

```yaml
- repo: https://github.com/thibaudcolas/pre-commit-stylelint
  rev: v14.4.0
  hooks:
    - id: stylelint
      additional_dependencies:
        # stylelint itself needs to be here when using additional_dependencies.
        - stylelint@14.4.0
        - stylelint-config-standard-scss@3.0.0
        # Package names starting with `@` need to be quoted.
        - "@scope/my-awesome-plugin@0.12.0"
```
